### PR TITLE
9663 - Add new APIs to validate and retrieve user infor from LTPA token

### PR DESCRIPTION
**Pull request by utle**
#build 
#run-libby-bot

### CODE REVIEW

File: dev/com.ibm.websphere.security/src/com/ibm/websphere/security/auth/ValidationFailedException.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.websphere.security.auth;
+
+/**
+ *
+ */
+public class ValidationFailedException extends InvalidTokenException {
```
Comment (tevans78) [inline dev/com.ibm.websphere.security/src/com/ibm/websphere/security/auth/ValidationFailedException.java]: If this is a public API then the source should have some good javadoc

Comment (utle) [reply dev/com.ibm.websphere.security/src/com/ibm/websphere/security/auth/ValidationFailedException.java]: done

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#run-libby-bot

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#run-libby-bot

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
#build 
#run-libby-bot

**Comment by utle** [comment]
PB for OL and CL : https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_tYiBgGXwEeqDwI5U1uUYvg

**Comment by utle** [comment]
#build

**Comment by utle** [comment]
PB for branch 9663-uniqueid-user-realm of OL and CL : https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_QhBzQGYHEeqDwI5U1uUYvg

https://wasrtc.hursley.ibm.com:9443/jazz/web/projects/WS-CD#action=com.ibm.team.build.viewResult&id=_O_FxsGjAEeqaXeM8BMnlag

**Comment by utle** [comment]
#run-libby-bot

**Pull request closed without merging**